stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -462,27 +462,12 @@ norm_sqr(x::Union{T,Complex{T},Rational{T}}) where {T<:Integer} = abs2(float(x))
 
 function generic_norm2(x)
     maxabs = normInf(x)
-    (maxabs == 0 || isinf(maxabs)) && return maxabs
-    (v, s) = iterate(x)::Tuple
+    (ismissing(maxabs) || maxabs == 0 || isinf(maxabs)) && return maxabs
```

Comment:
I'm sad if we need to add a bunch of special-cased `missing` code to LinearAlgebra.

Reply:
I know. We can leave out that test. It's just that the transition to `mapreduce` facilitated `norm` handle missing for certain `p`s, without pulling `*missing` to the surface.

Reply:
Would
```suggestion
    (maxabs == 0 || isinf(maxabs) !== false) && return maxabs
```
be preferable? (Which I think should be equivalent, but please do double-check.)

Reply:
So that you get a notification: I modified @martinholters's [suggestion](https://github.com/JuliaLang/julia/pull/43256#discussion_r760002436), which I tested for `maxabs = missing` and it works.

Reply:
Oh, and that same trick should make it possible to apply the same steps to `generic_normp`!